db/fixtures/miq_product_features.yml

```diff
@@ -6213,6 +6213,26 @@
       :feature_type: view
       :identifier: firmware_show
 
+# Guest Devices
```

Comment:
@dclarizio @Fryguy Do we want a `Guest Devices` section in the `product_features`?  I'm not sure of all the impact that has.